app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] [REQUIRED] Hello Team, you have done a great job with devise implementation 👍🏽 
However, the project requirement states that the views in devise for the registration and login should be modified to match the wireframes and their styling. Let's do this 🥇 

### Your design
<img width="464" alt="Screenshot 2024-01-27 at 21 38 44" src="https://github.com/Fombi-Favour/blog-app-rb/assets/58692590/24850ebd-d2fb-490b-b94c-15934fcb280a">

### Expected design
<img width="414" alt="Screenshot 2024-01-27 at 21 38 26" src="https://github.com/Fombi-Favour/blog-app-rb/assets/58692590/16483570-f738-4a45-8fea-97e9b169b24c">